spec/runners/unified/change_stream_operations.rb

```diff
@@ -37,5 +37,11 @@ def iterate_until_document_or_error(op)
       object = entities.get(:change_stream, object_id)
       object.to_enum.next
     end
+
+    def close(op)
+      object_id = op.use!('object')
+      object = entities.get(:change_stream, object_id)
```

Comment:
`close` is an operation on cursor, not change stream: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#close

Reply:
We do define a method close on change_stream: https://github.com/mongodb/mongo-ruby-driver/blob/master/lib/mongo/collection/view/change_stream.rb#L237

Reply:
oh I see, this was meant to go on the cursor... Okay I will switch it.